lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py

```diff
@@ -533,7 +533,7 @@ def cleanup(self, test=None):
 
         if not is_test_successful:
             print("The test has failed. Restarting Sctpd for cleanup")
-            self.magmad_util.restart_sctpd()
+            self.magmad_util.restart_services(['sctpd'], 30)
             self.magmad_util.print_redis_state()
```

Comment:
We could rename `restart_sctpd` to `_restart_sctpd`, to indicate that it should no longer be directly called from the tests. The way of doing this with `restart_services(['sctpd'], wait_time)` is not intuitive if a `restart_sctpd` is still public.
Same goes for `restart_mme`.